specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json

```diff
@@ -3103,6 +3223,204 @@
         }
       ],
       "description": "Represents set of actions written in SQL language-based syntax that is performed against a ServiceBus.Messaging.BrokeredMessage "
+    },
+    "PremiumMessagingRegions": {
+      "properties": {
+        "properties": {
+          "properties": {
+            "code": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Region code"
+            },
+            "fullName": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Full name of the region"
+            }
+          }
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ResourceNamespacePatch"
```

Comment:
A region resource without location property doesn't sound right, can you verify this.